.github/workflows/submit.yml

```diff
@@ -88,4 +89,6 @@ jobs:
 
       - name: Release
         if: ${{ !inputs.dryRun }}
-        run: pnpx changelogen@latest gh release ${{ steps.version.outputs.newVersion }} --token ${{ github.token }}
+        run: gh release create v${{ steps.version.outputs.newVersion }} .output/*.zip -F CHANGELOG.md
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
`changelogen gh release` just syncs the changelog with the latest release, it doesn't regenerate the changelog or anything like that. So these are doing almost the same thing.

The `-F CHANGELOG.md` is using your entire changelog as your release notes... Which I don't think you want?

I don't know why `github.token` worked for me and not for you, weird. You can replace it with `secrets.GITHUB_TOKEN` like you did, that's fine.

And uploading the zip files as artifacts would be a good idea...

Reply:
Sorry, I was confused with `changelogen gh release` command, not realizing it's a subcommand of changelogen package. So, it might be working correctly.